### PR TITLE
Update missing Homematic IP Components

### PR DESCRIPTION
**Pull request by SukramJ**
**Description:**
Updated missing Homematic IP Components.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23755

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by SukramJ** [comment]
Hi this PR does not depend on the next release. It's also valid for current.

File: source/_components/homematicip_cloud.markdown [hunk 1]
```diff
@@ -128,17 +136,28 @@ authtoken:
...
   * Multi IO Box - 2x (*HmIP-MIOB*)
 
 * homematicip_cloud.weather
   * Weather Sensor – basic (*HmIP-SWO-B*)
   * Weather Sensor – plus (*HmIP-SWO-PL*)
   * Weather Sensor – pro (*HmIP-SWO-PR*)
+  
+Additional info:
+
+Push button devices are only available with a battery sensor. This is due to a limitation of the ventor API (eq3).
```
Comment (MartinHjelmare) [inline source/_components/homematicip_cloud.markdown]: vendor API

File: source/_components/homematicip_cloud.markdown [hunk 2]
```diff
@@ -128,17 +136,28 @@ authtoken:
...
+It' not possible to detect a key press event on these devices at the moment.
```
Comment (MartinHjelmare) [inline source/_components/homematicip_cloud.markdown]: It's not possible

**Review by MartinHjelmare — APPROVED** [review]

**Comment by SukramJ** [comment]
Should this one not be in `current`?

**Comment by klaasnicolaas** [comment]
The changes from your parent PR will come in the 0.94 release and only then will the changes to the documentation also come online. If you adjusted things that didn't actually belong to this PR, you could have done this in a separate PR.

**Comment by SukramJ** [comment]
Thanks. My fault.

**Pull request merged**